velox/docs/functions/spark/math.rst

```diff
@@ -233,3 +233,7 @@ Mathematical Functions
 .. spark:function:: unaryminus(x) -> [same as x]
 
     Returns the negative of `x`.  Corresponds to Spark's operator ``-``.
+
+.. spark:function:: unhex(x) -> varbinary
+
+    Converts hexadecimal ``x`` to varbinary.
```

Comment:
Would you clarify the semantics when x is not well-formed, e.g. has odd number of characters or contains non-hex characters.

Reply:
enrich doc with two extra behaviors.